internal/pkg/agent/application/upgrade/artifact/download/fs/verifier_test.go

```diff
@@ -173,50 +173,60 @@ func prepareFetchVerifyTests(dropPath, targetDir, targetFilePath, hashTargetFile
 }
 
 func TestVerify(t *testing.T) {
-	log, _ := logger.New("", false)
-	targetDir, err := ioutil.TempDir(os.TempDir(), "")
-	if err != nil {
-		t.Fatal(err)
+	tt := []struct {
+		Name             string
+		RemotePGPUris    []string
+		UnreachableCount int
+	}{
+		{"default", nil, 0},
+		{"unreachable local path", []string{download.PgpSourceURIPrefix + "https://127.0.0.1:2874/path/does/not/exist"}, 1},
```

Comment:
It feels like this test could move to the top level download package now, instead of testing it specifically in the fs implementation.

Reply:
will add more unit tests for `PgpBytesFromSource` in a followup PR.

Reply:
Sure, the test coverage exists today it just isn't in the optimal location so I'm fine with that.